### PR TITLE
5323: Fix color of footer advice and size

### PR DESCRIPTION
**Pull request by kat1906**

### CODE REVIEW

File: app/pages/common/section/components/fields/text-field.njk [hunk 1]
```diff
@@ -58,7 +58,7 @@
       </div>
     {% endif %}
     <div data-test-id="text-field-footer">
-      <span class="nhsuk-label bc-c-field-footer">
+      <span class="nhsuk-label bc-t-font-color--grey-1 nhsuk-u-font-size-16r">
```
Comment (alibombaye) [inline app/pages/common/section/components/fields/text-field.njk]: `16r` is that `r` a typo?

File: app/pages/common/section/components/fields/textarea-field.njk [hunk 2]
```diff
@@ -60,7 +60,7 @@
       </div>
     {% endif %}
     <div data-test-id="textarea-field-footer">
-      <span class="nhsuk-label bc-c-field-footer">
+      <span class="nhsuk-label bbc-t-font-color--grey-1 nhsuk-u-font-size-16">
```
Comment (alibombaye) [inline app/pages/common/section/components/fields/textarea-field.njk]: `bbc` wrong company Katie 😄

File: app/pages/common/section/components/fields/text-field.njk (hunk 1 above)
Comment (kat1906) [reply app/pages/common/section/components/fields/text-field.njk]: 👀

File: app/pages/common/section/components/fields/textarea-field.njk (hunk 2 above)
Comment (kat1906) [reply app/pages/common/section/components/fields/textarea-field.njk]: lollll both these things were correct on v1

**Pull request merged**